### DISCUSSION TITLE
Recommended for Single Locale usage?

### DISCUSSION THREAD

**Discussion by thomasverleye** [sergiodxa/remix-i18next#64] [Q&A] [answered]
Hi 👋 

In the current project we're setting up I want to enable locale/translation files to use in combination with Lokalise.
Would it be overkill to use `remix-i18next` for just 1 locale?

It could happen that how the project grows other locales will be added but definitely not for the first half year.

Cheers,

Thomas

**Comment by sergiodxa** [comment, marked as answer]
I don’t think it’s an overkill, I use this at work for a single locale too so if we ever need to support more locales we can do it, adding i18n support to an app after the app was built is too much work IMO and it’s easy to forget texts to translate.

**Reply by thomasverleye** [reply]
Thanks, that makes sense 😉
